### PR TITLE
Update Cypress

### PR DESCRIPTION
**Pull request by benjaminpaige**
This change
- Replaces feature files with spec files
- removes all files related to feature files
- remove cucumber
- adds typescript
- misc cleanup in tests/cypress

### CODE REVIEW

**Review by abelzer212 — DISMISSED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -167,40 +167,40 @@ jobs:
...
+            DEVCH,
+            FUAAD,
+            FUHAD,
+            FUMAD,
+            FVAAD,
+            HPCAD,
+            HPCMIAD,
+            HVLAD,
+            MSCAD,
+            # NCIDDSAD,
```
Comment (benjaminpaige) [inline .github/workflows/deploy.yml:191]: why is this commented out ben?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -167,40 +167,40 @@ jobs:
...
+            OHDAD,
+            OUDAD,
+            PC01AD,
+            PCRAD,
+            PPCAD,
+            # PQI01AD,
```
Comment (benjaminpaige) [inline .github/workflows/deploy.yml:197]: ehem!

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (benjaminpaige) [reply .github/workflows/deploy.yml:191]: ahem!

**Review by jphechter — DISMISSED** [review]
🤝 👐 🙏 🙌

**Pull request merged**